docs/reference/api.md

```diff
@@ -14,35 +14,35 @@ For more examples, see "How to use Pebble API". <!-- [David] Link to the how-to
 
 To access the API endpoints over the Unix socket, use the `--unix-socket` option of `curl`. For example:
 
-* TODO
-
-    ```
-    curl ...
-    ```
+```{terminal}
+   :input: curl --unix-socket /path/to/.pebble.socket -XPOST http://localhost/v1/services -d '{"action": "stop", "services": ["svc1"]}'
+{"type":"async","status-code":202,"status":"Accepted","change":"42","result":null}
+```
 
-* TODO
+<br />
 
-    ```
-    curl ...
-    ```
+```{terminal}
+   :input: curl --unix-socket /path/to/.pebble.socket -XGET localhost:4000/v1/changes/42/wait
```

Comment:
I think we should drop `-XGET` as it's the default. Also, let's use `http://_/...` for consistency.